### DISCUSSION TITLE
[Help Needed] Need a new WALC logo

### DISCUSSION THREAD

**Discussion by cstayyab** [cstayyab/WALC#73] [General]
We need a new WALC Logo. (Example Idea: Combination of WhatsApp Logo and Linux Mascot (Tux) ).

Please submit your logo here: [http://bit.ly/WALCLogoSubmission](http://bit.ly/WALCLogoSubmission)

The name of all the participants will be added to a file in repository. **The candidate whose logo we end up using will be mentioned prominently in the README file as well as the About Dialog of WALC.**

Thank you for your time and effort. Keep using & supporting WALC

**Comment by tozman51** [comment]
I literally combined a tux inside whatsapp logo... :)

**Comment by Cediddi** [comment]
I turned whatsapp logo into a tux, looks awful because I'm bad at designing but this is my first inkscape design. Good luck
